pkg/controller/provider/ova-setup.go

```diff
@@ -92,15 +94,15 @@ func (r *Reconciler) createPvForNfs(provider *api.Provider, ctx context.Context,
 	if err != nil {
 		return
 	}
-	return
+	return pv.GetObjectMeta().GetName(), nil
```

Comment:
can be replaced with `pv.Name` but how about returning the PV instead? it makes more sense for a function named `createPvForNfs` in my opinion